homeassistant/components/knx/project.py

```diff
@@ -90,6 +91,7 @@ async def load_project(
         if project := data or await self._store.async_load():
             self.devices = project["devices"]
             self.info = project["info"]
+            GroupAddress.address_format = self.get_address_format()
```

Comment:
This looks incorrect. You are modifying a class object's value here. Ie the model object.

Reply:
Yes, a `ClassVar`. It's used by the xknx object for representing `GroupAddress.__str__()`